src/components/Callout/Callout.tsx

```diff
@@ -17,8 +18,18 @@ export const Callout = ({
       variation="filled"
       colorTheme={warning ? 'warning' : 'info'}
       backgroundColor={backgroundColor}
+      hasIcon={false}
     >
-      <View>{children}</View>
+      <Flex>
+        <div className="amplify-message__icon">
+          {warning ? (
+            <IconWarning aria-hidden={false} aria-label="warning icon" />
+          ) : (
+            <IconInfo aria-hidden={false} aria-label="info icon" />
+          )}
+        </div>
+        <div>{children}</div>
```

Comment:
```suggestion
        <div className="amplify-message__content">{children}</div>
```

I think this needs this class to prevent content from stretching. You can test on this page which currently has code snippets overflowing the warning Callout https://a11y-callout-icons.d1egzztxsxq9xz.amplifyapp.com/react/start/manual-installation/